etc/profile-a-l/googler-common.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for googler clones
+# Description: common profile for googler clones
+# This file is overwritten after every install/update
+# Persistent local customizations
+include googler-common.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}/wayland-*
```

Comment:
Can we `blacklist ${RUNUSER}` (everything, not just `wayland-*`) instead?

Reply:
ok